lib/node_modules/@stdlib/stats/base/smean/lib/ndarray.js

```diff
@@ -44,8 +44,8 @@ var smeanpn = require( '@stdlib/stats/base/smeanpn' ).ndarray;
 * var v = smean( N, x, 2, 1 );
```

Comment:
```suggestion
* var v = smean( 4, x, 2, 1 );
```
kindly remove the import for `floor` as well